twyper/src/main/java/com/github/theapache64/twyper/CardController.kt

```diff
@@ -21,6 +19,10 @@ interface CardController {
     val cardX: Float
     val cardY: Float
     val rotation: Float
+    val cardFlipRotation: Float
+    val cardFrontAlpha: Float
+    val cardBackAlpha: Float
+    var state: FlipState
     var swipedOutDirection: SwipedOutDirection?
```

Comment:
I think this should be inside a separate class. Consider create a new class by subclassing `CardControllerImpl`. The problem is, here we are considering all cards as flippable. But that's not true. Only cards that are inside TwyperFlip is flippable.